resources/docs/html_en_EN/scripteditor.html

```diff
@@ -231,6 +231,30 @@ <H3>Language syntax</H3>
 wait:03s.20 // Waits for 3 seconds and 200 milliseconds</PRE>
   </TD>
  </TR>
+ <TR>
```

Comment:
Docs in HTML form are no longer bundled and maintained.
Please remove this all and submit a PR for this
https://github.com/mcallegari/qlcplus-docs/blob/main/pages/04.function-manager/08.script-editor/default.v4.md

Reply:
Documentation moved to PR https://github.com/mcallegari/qlcplus-docs/pull/42